docs/manual/comparison_with_pandas.rst

```diff
@@ -0,0 +1,246 @@
+=================================================  ============================================
+Select a single row
+``df.loc[2]``                                        ``DT[2, :]``
+
+Select ``n`` rows
+``df.iloc[[2,3,4]]``                                  ``DT[[2,3,4], :]``
+
+Select a slice of rows by position
+``df.iloc[range(2,5)]``                               ``DT[range(2,5), :]``
+
+Same as above
+``df.iloc[2:5]``                                      ``DT[2:5, :]``
+
+Select every ``n`` rows
+``df.iloc[::2]``                                       ``DT[::2, :]``
+
+Select rows based on booleans
+``df.iloc[[True, True, False, False, True]]``            ``DT[[True, True, False, False, True], :]``
+
+Select rows on a condition
+``df.loc[df['A']>3]``                                   ``DT[f.A>3, :]``
+
+Select rows on multiple conditions, using ``OR``
+``df.loc[(df['A'] > 3) | (df['B']<5)]``                   ``DT[(f.A>3) | (f.B<5), :]``
+
+Select rows on multiple conditions, using ``AND``
+``df.loc[(df['A'] > 3) & (df['B']<8)]``                  ``DT[(f.A>3) & (f.B<8), :]``
+
+Select a single column by column name
+       ``df['A']``                                     ``DT['A']``
+
+Same as above
+       ``df.loc[:, 'A']``                              ``DT[:, 'A']``
+
+Select a single column by position
+``df.iloc[:, 1]``                                       ``DT[1]`` or ``DT[:, 1]``
+
+Select multiple columns by column names
+``df.loc[:, ["A", "B"]]``                              ``DT[:, ["A", "B"]]``
+
+Select multiple columns by position
+``df.iloc[:, [0, 1]]``                                ``DT[:, [0, 1]]``
+
+Select multiple columns by slicing
+``df.loc[:, "A":"B"]``                                 ``DT[:, "A":"B"]``
+
+Same as above, by position
+``df.iloc[:, 1:3]``                                      ``DT[:, 1:3]``
+
+Select columns by boolean
+``df.loc[:, [True,False,False,True]]``                ``DT[:, [True,False,False,True]]``
+
+Select multiple rows and columns
+``df.loc[2:5, "A":"B"]``                              ``DT[2:5, "A":"B"]``
+
+Same as above, by position
+``df.iloc[2:5, :2]``                                    ``DT[2:5, :2]``
+
+Select a single value (returns a scalar)
+``df.at[2, 'A']`` or ``df.loc[2, 'A']``                 ``DT[2,"A"]``
+
+Same as above, by position
+``df.iat[2, 0]``  or  ``df.iloc[2, 0]``                 ``DT[2, 0]``
+
+Select a single value, return as Series                Returns a Frame
+``df.loc[2, ["A"]]``                                  ``DT[2, ["A"]]``
+
+Same as above, by position
+``df.iloc[2, [0]]``                                  ``DT[2, [0]]``
+=================================================  ============================================
+
+
+In ``pandas`` the index numbers are returned :
+
+.. code-block:: python
+
+    # pandas
+    df.loc[df['A']>3]
+
+    	A	B	C	D
+    3	4	7	10	9
+    4	5	8	11     -1
+
+This is not the case in ``datatable`` :
```

Comment:
The actual distinction here is that in pandas every frame has row Index, a separate object which records the id of each observation, much like rownames in R dataframes. 
In datatable, however, there is no such object. The row numbers displayed in the terminal output are just for convenience, but they are not stored as a separate object.

If a person needs the equivalent of python row index, they can simply create a new column with `range(DT.nrows)` and then do whatever they want with that column, including filtering etc.